lib/awful/mouse/init.lua

```diff
@@ -326,6 +325,19 @@ end
 
 function mouse.object.set_current_widget() end
 
+--- Get the current widget geometry.
+-- @property current_widget_geometry
+-- @tparam ?table The geometry.
```

Comment:
properties documentation is using param. ldoc is configured to turn that into a "type" section. I could create an alias to make that less confusing...